Add metadata fields to openapi-2-kong package.json

mikaello opened the pull request:
The metadata fields on https://www.npmjs.com/package/openapi-2-kong are a bit scarce, and it would be nice if these were improved:

![image](https://user-images.githubusercontent.com/2505178/97800293-597a5a80-1c34-11eb-9705-e236e669a9e9.png)

mikaello committed:
add metadata fields to openapi-2-kong package.json

mikaello committed:
Merge branch 'develop' into patch-1

develohpanda commented on packages/openapi-2-kong/package.json:
```diff
@@ -29,5 +29,8 @@
     "webpack-cli": "^3.3.11"
   },
+  "repository": "https://github.com/Kong/insomnia/tree/develop/packages/openapi-2-kong",
+  "bugs": "https://github.com/Kong/insomnia/issues",
```
According to [the docs](https://docs.npmjs.com/cli/v6/configuring-npm/package-json#bugs), this needs to be the following:

```suggestion
  "bugs": {
    "url": "https://github.com/Kong/insomnia/issues"
  },
```

mikaello replied:
Thanks for looking up! I based my changes on an old convention, obviously outdated. Fixed now.

develohpanda commented on packages/openapi-2-kong/package.json:
```diff
@@ -29,5 +29,8 @@
   },
+  "repository": "https://github.com/Kong/insomnia/tree/develop/packages/openapi-2-kong",
+  "bugs": "https://github.com/Kong/insomnia/issues",
+  "license": "MIT",
```
@nijikokun should the license be MIT or Apache2? Inso is set to Apache2.

mikaello replied:
Just a comment to this question: if you set this to Apache2, I think you should change the licence that is distributed with openapi-2-kong, which is currently MIT: 

```shell
$ npm pack openapi-2-kong
$ tar xzf openapi-2-kong-2.2.24.tgz
$ cat package/LICENSE

MIT License

Copyright (c) Insomnia REST Client

Permission is hereby granted....
```

For reference why this came to my attention in the first place, see [discussion in Kong Nation](https://discuss.konghq.com/t/import-swagger-open-api-document-to-create-api-in-kong/510/18?u=michael_johansen).

develohpanda replied:
A [recent PR](https://github.com/Kong/insomnia/pull/3233) went in to set up license fields using feedback from the Legal team, so we can remove this field from this PR. I have created a separate internal ticket INS-658 in order to address your suggestion (with this package and all other packages).

```suggestion
```

develohpanda reviewed (commented):
Thanks for doing this! One comment, and a question for @nijikokun

mikaello committed:
fix issue url in openapi-2-kong package.json

Format according to npm docs: https://docs.npmjs.com/cli/v6/configuring-npm/package-json#bugs

develohpanda reviewed (approved):
🎉

mikaello committed:
Merge branch 'develop' into patch-1

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/Kong/insomnia?pullRequest=2802) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/Kong/insomnia?pullRequest=2802) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/Kong/insomnia?pullRequest=2802) it.</sub>

mikaello committed:
Merge branch 'develop' into patch-1

mikaello committed:
Merge branch 'develop' into patch-1

mikaello committed:
improve repo-field in openapi-2-kong package.json

See structure in NPM docs: https://docs.npmjs.com/cli/v6/configuring-npm/package-json#repository

develohpanda committed:
Merge branch 'develop' into patch-1

develohpanda commented on packages/openapi-2-kong/package.json:
```diff
@@ -31,5 +31,14 @@
     "webpack": "^4.42.1",
     "webpack-cli": "^3.3.11"
   },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Kong/insomnia.git",
+    "directory": "packages/openapi-2-kong"
+  },
```
Nice improvement here! We should certainly update the other packages to follow this pattern, TIL.

develohpanda committed:
Update packages/openapi-2-kong/package.json

reynolek reviewed (approved):
Approved from product perspective
